src/shapes/Image.ts

```diff
@@ -856,5 +851,8 @@ export class Image<
   }
 }
 
-classRegistry.setClass(Image);
-classRegistry.setSVGClass(Image);
+classRegistry.setClass(FabricImage);
+classRegistry.setSVGClass(FabricImage);
+
+/** @deprecated Image is a keyword. Please use FabricImage instead */
```

Comment:
i would extend this:

WARNING the old fabric.Image class can't be imported as Image because of a reserved word clash.
For this reason it has been renamed to FabricImage. Please use FabricImage in place of Image.